packages/aws-cdk/lib/cli/cdk-toolkit.ts

```diff
@@ -359,6 +360,24 @@ export class CdkToolkit {
         );
       }
 
+      const currentTemplate = await this.props.deployments.readCurrentTemplate(stack);
```

Comment:
This will need to be added to the new Toolkit as well (yes I now it's annoying).